pdns/stubquery.cc

```diff
@@ -38,19 +38,21 @@ try
     }
   }
 
-  if(argc < 2) {
+  if(argc <= 1) {
     usage();
     exit(EXIT_FAILURE);
   }
 
+  string type(argc == 2 ? "A" : argv[2]);
```

Comment:
```suggestion
  string type(argc >= 2 ? "A" : argv[2]);
```

Reply:
I must be missing something. We want to take arg[2] if it is given.

Reply:
Oh, my logic is reversed! I was thinking about what if we later add more arguments (like sdig flags, although we should really redo those in a better way).

My suggestion should have been `<= 2`, I suppose, but let's drop it.